src/controllers/business/index.js

```diff
@@ -0,0 +1,59 @@
+export const getRealName = async (obj) => {
+  const legalInformation = await obj.getLegalInformation()
+
+  return legalInformation.businessName
+}
+
+export const getVertical = async (obj) => {
+  const legalInformation = await obj.getLegalInformation()
+
+  return legalInformation.businessVertical
+}
+
+export const getCUIT = async (obj) => {
+  const legalInformation = await obj.getLegalInformation()
+
+  return legalInformation.CUIT
+}
+
+export const getAFIPCondition = async (obj) => {
+  const legalInformation = await obj.getLegalInformation()
+
+  return legalInformation.AFIPCondition
+}
+
+export const getAddress = (obj) => obj.getRealAddress()
+
+export const createBusiness = (obj, { input }, { models }) => {
+  return models.sequelize.transaction(async (transaction) => {
+    const {
+      fantasyName,
+      realName: businessName,
+      vertical: businessVertical,
+      CUIT,
+      AFIPCondition,
+      address: addressInput
+    } = input
+
+    const address = await models.physicalAddress.create(addressInput, { transaction })
+
+    const legalInformation = await models.legalInformation.create({
+      businessName,
+      businessVertical,
+      CUIT: Number(CUIT),
+      AFIPCondition,
+      physicalAddressId: address.id
+    }, { transaction })
+
+    const business = await models.business.create({
+      fantasyName,
+      physicalAddressId: address.id
+    }, { transaction })
+
+    await business.setLegalInformation(legalInformation, { transaction })
+
+    return business
+  })
+}
+
+export const getBusinessById = (obj, { id }, { models }) => models.business.findByPk(id)
```

Comment:
You're returning `null` if the given `id` doesn't exist in the database, but in GraphQL you defined the query to be not `null`.

You should either:
- throw an error in this resolver if a business was not found with the given `id`, or
- change the schema to be able to return `null`.

I prefer the first option, throwing an error that contains an error code (to be able to distinguish it in the front-end) and a descriptive message.